lib/index.js

```diff
@@ -118,6 +143,17 @@ class Unifile {
 		this[connectors].set(connector.name.toLowerCase(), connector);
 	}
 
+	/**
+   * Adds an extension into Unifile.
+   * The plugin actions will be applied for all the connectors.
+   * @param {Plugin} plugin - A plugin with at least one hook implemented
+   */
+	ext(plugin) {
+		Object.keys(HOOKS).map((key) => HOOKS[key]).forEach((hook) => {
```

Comment:
Object.values(HOOKS) ?

Reply:
Better, I was looking for this but got stuck with forof loop